pkg/apis/kops/registry/registry.go

```diff
@@ -29,6 +29,8 @@ const (
 	PathCluster = "config"
 	// Path for completed cluster spec in the state store
 	PathClusterCompleted = "cluster.spec"
+	// PathKopsVersion is the path for the version of kops last used to apply the cluster.
+	PathKopsVersionApplied = "kops-version-applied.txt"
```

Comment:
```suggestion
	// PathKopsVersionUsed is the path for the version of kops last used to apply the cluster.
	PathKopsVersionUsed = "kops.version"
```

Reply:
Again, it's not the last kops version used.